<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AAWidget</class>
 <widget class="QWidget" name="AAWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>198</width>
     <height>187</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_main">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_val">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">font-family:WenQuanYi Micro Hei;font-size:60px;color:green</string>
          </property>
          <property name="text">
           <string notr="true">-20</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_max">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {color:white;}</string>
          </property>
          <property name="text">
           <string notr="true">max</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {color:white;}</string>
          </property>
          <property name="text">
           <string notr="true">dBμV</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="2" column="2">
       <widget class="QLabel" name="label_50Hz">
        <property name="styleSheet">
         <string notr="true">QLabel {color:white;}</string>
        </property>
        <property name="text">
         <string notr="true">20mV</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QProgressBar" name="progressBar_effective">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>15</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QProgressBar{border:1px solid gray;background:transparent;}
QProgressBar::chunk{&quot;background-color:skyblue;&quot;}</string>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QProgressBar" name="progressBar_50Hz">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>15</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QProgressBar{border:1px solid gray;background:transparent;}
QProgressBar::chunk{&quot;background-color:skyblue;&quot;}</string>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QProgressBar" name="progressBar_100Hz">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>15</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QProgressBar{border:1px solid gray;background:transparent;}
QProgressBar::chunk{&quot;background-color:skyblue;&quot;}</string>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_100Hz">
        <property name="styleSheet">
         <string notr="true">QLabel {color:white;}</string>
        </property>
        <property name="text">
         <string notr="true">10mV</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_peak">
        <property name="styleSheet">
         <string notr="true">QLabel {color:white;}</string>
        </property>
        <property name="text">
         <string notr="true">20mV</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_range_2">
        <property name="styleSheet">
         <string notr="true">QLabel {color:white;}</string>
        </property>
        <property name="text">
         <string notr="true">100Hz</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QProgressBar" name="progressBar_peak">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>15</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QProgressBar{border:1px solid gray;background:transparent;}
QProgressBar::chunk{&quot;background-color:skyblue;&quot;}</string>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_range_4">
        <property name="styleSheet">
         <string notr="true">QLabel {color:white;}</string>
        </property>
        <property name="text">
         <string>峰值</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_effective">
        <property name="styleSheet">
         <string notr="true">QLabel {color:white;}</string>
        </property>
        <property name="text">
         <string notr="true">20mV</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_range">
        <property name="styleSheet">
         <string notr="true">QLabel {color:white;}</string>
        </property>
        <property name="text">
         <string notr="true">50Hz</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_range_3">
        <property name="styleSheet">
         <string notr="true">QLabel {color:white;}</string>
        </property>
        <property name="text">
         <string>有效值</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
